### DISCUSSION TITLE
Can't Connect to Postgres AWS RDS

### DISCUSSION THREAD

**Discussion by erasebegin** [prisma/prisma#15892] [Q&A] [answered]
Hi, I'm having trouble connecting to my database using the following url in my .env:
```
DATABASE_URL="postgresql://<username>:<password>@<db-name>.clpqhwqh8vk6.eu-west-2.rds.amazonaws.com:5432/<db-name>"
```

When executing `npx prisma migrate dev --name init` I get the following error:
```
Error: P1001: Can't reach database server at `frivle-db-2.7h4eyryuyh65e.eu-west-2.rds.amazonaws.com`:`5432`
```
I have tried so many different combinations and gone through the documentation multiple times, created several new db instances, but nothing has worked. Please help 🙏🙏🙏

**Comment by janpio** [comment, marked as answer]
Can you connect with other tools from your local machine? RDS instances (which I assume you meant with `WDS` in the title) by default are not reachable from the internet and need to be configured that way.

**Reply by erasebegin** [reply]
That's a really good point, thank you! I came from Atlas which enables this by default so I made a poor assumption.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
